### PR TITLE
Build on mac mojave

### PR DESCRIPTION
**Pull request by wlach**

### CODE REVIEW

**Comment by wlach** [comment]
We should also enable CI on Mac, so this doesn't break, this depends on a paid plan so on the resolution of https://bugzilla.mozilla.org/show_bug.cgi?id=1589709

**Comment by wlach** [comment]
@mdboom it is unclear whether this is causing new failures based on our discussion (I suspect it isn't), but regardless I think this is ready for at least a preliminary review.

File: README.md [hunk 1]
```diff
@@ -64,6 +65,15 @@ Additional build prerequisites are:
...
 - [ccache](https://ccache.samba.org) (optional) recommended for much faster rebuilds.
 
+On Mac, you will also need:
+
+- [Homebrew](https://brew.sh/) for installing dependencies
+- System libraries in the root directory (`sudo installer -pkg /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg -target /` should do it, see https://github.com/pyenv/pyenv/issues/1219#issuecomment-428305417)
+- coreutils for md5sum and other essential Unix utilities (`brew install coreutils`)
+- openssl (`brew install openssl`)
+- gfortran (`brew cask install gfortran`)
+- f2c: Install wget (`brew install wget`), and then download and run this gist as root: https://gist.github.com/wlach/b5edcd895496298074e550952f0fe3ef
```
Comment (mdboom) [inline README.md]: Should we just include this gist in the `tools` directory here?

File: README.md [hunk 2]
```diff
@@ -75,7 +85,9 @@ make
 ### Using Docker
 
 We provide a Debian-based Docker image on Docker Hub with the dependencies
-already installed to make it easier to build Pyodide.
+already installed to make it easier to build Pyodide. Note that building from
+the Docker image is *very* slow on Mac, building on the host machine is
```
Comment (mdboom) [inline README.md]: `on Mac` -> `non-Linux hosts`?

Comment (wlach) [reply README.md]: Is Docker for Windows actually all that slow?

File: README.md (hunk 1 above)
Comment (wlach) [reply README.md]: Makes sense to me, will address before landing

Comment (wlach) [reply README.md]: Would of course be nice to be able to skip this step, maybe something to address in a followup.

File: README.md (hunk 2 above)
Comment (wlach) [reply README.md]: Let's punt on this for now, the wording is ok as-is IMO. At some point in the future maybe a Windows-user can add some more notes for their platform.

**Pull request merged**